data/json/items/tool/cooking.json

```
@@ -860,6 +860,27 @@
     },
     "melee_damage": { "bash": 4 }
   },
+  {
+    "id": "snow_machine_home",
+    "type": "TOOL",
+    "name": { "str": "home snow machine" },
+    "description": "A snow machine, which freezes water and shreds it into \"snow\", designed to be used from the comfort of your own home!  Useful for making snowcones.",
```

```suggestion
    "description": "A snow machine, which freezes water and shreds it into \"snow\", designed to be used from the comfort of your own home!  Useful for making snow cones.",
```

You're gonna wanna add a space here to please the spellchecker.